src/SerialFirmata.cpp

```diff
@@ -66,6 +66,11 @@ boolean SerialFirmata::handleSysex(byte command, byte argc, byte *argv)
                 // because all Arduino pins are set to OUTPUT by default in StandardFirmata.
                 pinMode(pins.rx, INPUT);
               }
+#if defined(SERIAL_STORE_AND_FORWARD)    
```

Comment:
done, commit pending
Note: addressing a serial port above 15 will result in an array bounds violation, we could add a check in line 51.